File: Portal SAE/backend/src/main/java/com/siscem/portal_sae/dtos/email/EmailDTO.java
package com.siscem.portal_sae.dtos.email;

import java.util.Date;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.Data;

@Data
public class EmailDTO {
	private String codigo;
	private String remitente;
	private String destinatario;
	private String asunto;
	private String contenido;
	@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
	private Date fechaEnviado;
	@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
	private Date fechaRecibido;
	private String eventoAsignado;
}

File: Portal SAE/frontend/src/app/interfaces/email/email.dto.ts
export interface EmailDTO {
  codigo: string;
  remitente: string;
  destinatario: string;
  asunto: string;
  contenido: string;
  fechaEnviado: string;
  fechaRecibido: string;
  eventoAsignado: number;
  adjunto : File;
}
